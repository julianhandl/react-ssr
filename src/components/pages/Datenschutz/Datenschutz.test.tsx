import Datenschutz from "./Datenschutz";
import { mount } from "enzyme";
import React from "react";

describe("Datenschutz", () => {
    const wrapper = mount(<div>
        <Datenschutz key="datenschutz" name="datenschutz" contentTitle="Datenschutz" metaTitle="Datenschutz | Party Partner" metaDescription="" />
    </div>)

    it("should render correctly", () => {
        expect(wrapper).toMatchSnapshot();
    });
});