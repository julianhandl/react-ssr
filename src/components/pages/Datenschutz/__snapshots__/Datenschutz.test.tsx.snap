// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Datenschutz should render correctly 1`] = `
<div>
  <Datenschutz
    contentTitle="Datenschutz"
    key="datenschutz"
    metaDescription=""
    metaTitle="Datenschutz | Party Partner"
    name="datenschutz"
  >
    <HelmetWrapper
      defer={true}
      encodeSpecialCharacters={true}
      key="helmet-key-datenschutz"
    >
      <SideEffect(NullComponent)
        defer={true}
        encodeSpecialCharacters={true}
        meta={
          Array [
            Object {
              "content": "",
              "name": "description",
            },
          ]
        }
        title="Datenschutz | Party Partner"
        titleAttributes={Object {}}
      >
        <NullComponent
          defer={true}
          encodeSpecialCharacters={true}
          meta={
            Array [
              Object {
                "content": "",
                "name": "description",
              },
            ]
          }
          title="Datenschutz | Party Partner"
          titleAttributes={Object {}}
        />
      </SideEffect(NullComponent)>
    </HelmetWrapper>
    <div
      className="container"
      key="page-datenschutz-key"
    >
      <section
        className="page content-page page--datenschutz"
      >
        <h1
          className="content-page__title"
        >
          Datenschutz
        </h1>
        <div
          className="content-page__text"
        >
          <h3>
            Datenschutz
          </h3>
          Wir haben diese Datenschutzerklärung (Fassung 31.05.2019-221094977) verfasst, um Ihnen gemäß der Vorgaben der Datenschutz-Grundverordnung (EU) 2016/679 und dem Datenschutzgesetz (DSG) zu erklären, welche Informationen wir sammeln, wie wir Daten verwenden und welche Entscheidungsmöglichkeiten Sie als Besucher dieser Webseite haben. Leider liegt es in der Natur der Sache, dass diese Erklärungen sehr technisch klingen, wir haben uns bei der Erstellung jedoch bemüht die wichtigsten Dinge so einfach und klar wie möglich zu beschreiben. Automatische Datenspeicherung Wenn Sie heutzutage Webseiten besuchen, werden gewisse Informationen automatisch erstellt und gespeichert, so auch auf dieser Webseite. Wenn Sie unsere Webseite so wie jetzt gerade besuchen, speichert unser Webserver (Computer auf dem diese Webseite gespeichert ist) automatisch Daten wie
          <ul>
            <li>
              die Adresse (URL) der aufgerufenen Webseite
            </li>
            <li>
              Browser und Browserversion
            </li>
            <li>
              das verwendete Betriebssystem
            </li>
            <li>
              die Adresse (URL) der zuvor besuchten Seite (Referrer URL)
            </li>
            <li>
              den Hostname und die IP-Adresse des Geräts von welchem aus zugegriffen wird
            </li>
            <li>
              Datum und Uhrzeit
            </li>
          </ul>
          in Dateien (Webserver-Logfiles). In der Regel werden Webserver-Logfiles zwei Wochen gespeichert und danach automatisch gelöscht. Wir geben diese Daten nicht weiter, können jedoch nicht ausschließen, dass diese Daten beim Vorliegen von rechtswidrigem Verhalten eingesehen werden. Die Rechtsgrundlage besteht nach Artikel 6  Absatz 1 f DSGVO (Rechtmäßigkeit der Verarbeitung) darin, dass berechtigtes Interesse daran besteht, den fehlerfreien Betrieb dieser Webseite durch das Erfassen von Webserver-Logfiles zu ermöglichen.
          <h3>
            Cookies
          </h3>
          Unsere Website verwendet HTTP-Cookies um nutzerspezifische Daten zu speichern. Im Folgenden erklären wir, was Cookies sind und warum Sie genutzt werden, damit Sie die folgende Datenschutzerklärung besser verstehen.
          <h4>
            Was genau sind Cookies?
          </h4>
          Immer wenn Sie durch das Internet surfen, verwenden Sie einen Browser. Bekannte Browser sind beispielsweise Chrome, Safari, Firefox, Internet Explorer und Microsoft Edge. Die meisten Webseiten speichern kleine Text-Dateien in Ihrem Browser. Diese Dateien nennt man Cookies. Eines ist nicht von der Hand zu weisen: Cookies sind echt nützliche Helferlein. Fast alle Webseiten verwenden Cookies. Genauer gesprochen sind es HTTP-Cookies, da es auch noch anderer Cookies für andere Anwendungsbereiche gibt. HTTP-Cookies sind kleine Dateien, die von unserer Website auf Ihrem Computer gespeichert werden. Diese Cookie-Dateien werden automatisch im Cookie-Ordner, quasi dem „Hirn“ Ihres Browsers, untergebracht. Ein Cookie besteht aus einem Namen und einem Wert. Bei der Definition eines Cookies müssen zusätzlich ein oder mehrere Attribute angegeben werden. Cookies speichern gewisse Nutzerdaten von Ihnen, wie beispielsweise Sprache oder persönliche Seiteneinstellungen. Wenn Sie unsere Seite wieder aufrufen, übermittelt Ihr Browser die „userbezogenen“ Informationen an unsere Seite zurück. Dank der Cookies weiß unsere Website, wer Sie sind und bietet Ihnen Ihre gewohnte Standardeinstellung. In einigen Browsern hat jedes Cookie eine eigene Datei, in anderen wie beispielsweise Firefox sind alle Cookies in einer einzigen Datei gespeichert. Es gibt sowohl Erstanbieter Cookies als auch Drittanbieter-Cookies. Erstanbieter-Cookies werden direkt von unserer Seite erstellt, Drittanbieter-Cookies werden von Partner-Webseiten (z.B. Google Analytics) erstellt. Jedes Cookie ist individuell zu bewerten, da jedes Cookie andere Daten speichert. Auch die Ablaufzeit eines Cookies variiert von ein paar Minuten bis hin zu ein paar Jahren. Cookies sind keine Software-Programme und enthalten keine Viren, Trojaner oder andere „Schädlinge“. Cookies können auch nicht auf Informationen Ihres PCs zugreifen. So können zum Beispiel Cookie-Daten aussehen: Name: _ga Ablaufzeit: 2 Jahre Verwendung: Unterscheidung der Webseitenbesucher Beispielhafter Wert: GA1.2.1326744211.152221094977 Ein Browser sollte folgende Mindestgrößen unterstützen: Ein Cookie soll mindestens 4096 Bytes enthalten können Pro Domain sollen mindestens 50 Cookies gespeichert werden können Insgesamt sollen mindestens 3000 Cookies gespeichert werden können
          <h4>
            Welche Arten von Cookies gibt es?
          </h4>
          Die Frage welche Cookies wir im Speziellen verwenden, hängt von den verwendeten Diensten ab und wird in der folgenden Abschnitten der Datenschutzerklärung geklärt. An dieser Stelle möchten wir kurz auf die verschiedenen Arten von HTTP-Cookies eingehen. Man kann 4 Arten von Cookies unterscheiden:
          <h5>
            Unbedingt notwendige Cookies
          </h5>
          Diese Cookies sind nötig, um grundlegende Funktionen der Website sicherzustellen. Zum Beispiel braucht es diese Cookies, wenn ein User ein Produkt in den Warenkorb legt, dann auf anderen Seiten weitersurft und später erst zur Kasse geht. Durch diese Cookies wird der Warenkorb nicht gelöscht, selbst wenn der User sein Browserfenster schließt.
          <h5>
            Funktionelle Cookies
          </h5>
          Diese Cookies sammeln Infos über das Userverhalten und ob der User etwaige Fehlermeldungen bekommt. Zudem werden mithilfe dieser Cookies auch die Ladezeit und das Verhalten der Website bei verschiedenen Browsern gemessen.
          <h5>
            Zielorientierte Cookies
          </h5>
          Diese Cookies sorgen für eine bessere Nutzerfreundlichkeit. Beispielsweise werden eingegebene Standorte, Schriftgrößen oder Formulardaten gespeichert.
          <h5>
            Werbe-Cookies
          </h5>
          Diese Cookies werden auch Targeting-Cookies genannt. Sie dienen dazu dem User individuell angepasste Werbung zu liefern. Das kann sehr praktisch, aber auch sehr nervig sein. Üblicherweise werden Sie beim erstmaligen Besuch einer Webseite gefragt, welche dieser Cookiearten Sie zulassen möchten. Und natürlich wird diese Entscheidung auch in einem Cookie gespeichert.
          <h4>
            Wie kann ich Cookies löschen?
          </h4>
          Wie und ob Sie Cookies verwenden wollen, entscheiden Sie selbst. Unabhängig von welchem Service oder welcher Website die Cookies stammen, haben Sie immer die Möglichkeit Cookies zu löschen, nur teilweise zuzulassen oder zu deaktivieren. Zum Beispiel können Sie Cookies von Drittanbietern blockieren, aber alle anderen Cookies zulassen. Falls Sie grundsätzlich keine Cookies haben wollen, können Sie Ihren Browser so einrichten, dass er Sie immer informiert, wenn ein Cookie gesetzt werden soll. So können Sie bei jedem einzelnen Cookie entscheiden, ob Sie das Cookie erlauben oder nicht. Die Vorgangsweise ist je nach Browser verschieden. Am besten ist es Sie suchen die Anleitung in Google mit dem Suchbegriff “Cookies löschen Chrome” oder „Cookies deaktivieren Chrome“ im Falle eines Chrome Browsers oder tauschen das Wort „Chrome“ gegen den Namen Ihres Browsers, z.B. Edge, Firefox, Safari aus.
          <h4>
            Wie sieht es mit meinem Datenschutz aus?
          </h4>
          Seit 2009 gibt es die sogenannten „Cookie-Richtlinien“. Darin ist festgehalten, dass das Speichern von Cookies eine Einwilligung des Website-Besuchers (also von Ihnen) verlangt. Innerhalb der EU-Länder gibt es allerdings noch sehr unterschiedliche Reaktionen auf diese Richtlinien. In Österreich erfolgte aber die Umsetzung dieser Richtlinie in § 96 Abs. 3 des Telekommunikationsgesetzes (TKG). Wenn Sie mehr über Cookies wissen möchten und vor technischen Dokumentationen nicht zurückscheuen, empfehlen wir https://tools.ietf.org/html/rfc6265, dem Request for Comments der Internet Engineering Task Force (IETF) namens „HTTP State Management Mechanism“.
          <h3>
            Speicherung persönlicher Daten
          </h3>
          Persönliche Daten, die Sie uns auf dieser Website elektronisch übermitteln, wie zum Beispiel Name, E-Mail-Adresse, Adresse oder andere persönlichen Angaben im Rahmen der Übermittlung eines Formulars oder Kommentaren im Blog, werden von uns gemeinsam mit dem Zeitpunkt und der IP-Adresse nur zum jeweils angegebenen Zweck verwendet, sicher verwahrt und nicht an Dritte weitergegeben. Wir nutzen Ihre persönlichen Daten somit nur für die Kommunikation mit jenen Besuchern, die Kontakt ausdrücklich wünschen und für die Abwicklung der auf dieser Webseite angebotenen Dienstleistungen und Produkte. Wir geben Ihre persönlichen Daten ohne Zustimmung nicht weiter, können jedoch nicht ausschließen, dass diese Daten beim Vorliegen von rechtswidrigem Verhalten eingesehen werden. Wenn Sie uns persönliche Daten per E-Mail schicken – somit abseits dieser Webseite – können wir keine sichere Übertragung und den Schutz Ihrer Daten garantieren. Wir empfehlen Ihnen, vertrauliche Daten niemals unverschlüsselt per E-Mail zu übermitteln. Die Rechtsgrundlage besteht nach Artikel 6  Absatz 1 a DSGVO (Rechtmäßigkeit der Verarbeitung) darin, dass Sie uns die Einwilligung zur Verarbeitung der von Ihnen eingegebenen Daten geben. Sie können diesen Einwilligung jederzeit widerrufen – eine formlose E-Mail reicht aus, Sie finden unsere Kontaktdaten im Impressum.
          <h3>
            Auswertung des Besucherverhaltens
          </h3>
          In der folgenden Datenschutzerklärung informieren wir Sie darüber, ob und wie wir Daten Ihres Besuchs dieser Website auswerten. Die Auswertung der gesammelten Daten erfolgt in der Regel anonym und wir können von Ihrem Verhalten auf dieser Website nicht auf Ihre Person schließen. Mehr über Möglichkeiten dieser Auswertung der Besuchsdaten zu widersprechen erfahren Sie in der folgenden Datenschutzerklärung.
          <h3>
            TLS-Verschlüsselung mit https
          </h3>
          Wir verwenden https um Daten abhörsicher im Internet zu übertragen (Datenschutz durch Technikgestaltung Artikel 25 Absatz 1 DSGVO). Durch den Einsatz von TLS (Transport Layer Security), einem Verschlüsselungsprotokoll zur sicheren Datenübertragung im Internet können wir den Schutz vertraulicher Daten sicherstellen. Sie erkennen die Benutzung dieser Absicherung der Datenübertragung am kleinen Schlosssymbol links oben im Browser und der Verwendung des Schemas https (anstatt http) als Teil unserer Internetadresse.
          <h3>
            Newsletter Datenschutzerklärung
          </h3>
          Wenn Sie sich für unseren Newsletter eintragen übermitteln Sie die oben genannten persönlichen Daten und geben uns das Recht Sie per E-Mail zu kontaktieren. Die im Rahmen der Anmeldung zum Newsletter gespeicherten Daten nutzen wir ausschließlich für unseren Newsletter und geben diese nicht weiter. Sollten Sie sich vom Newsletter abmelden – Sie finden den Link dafür in jedem Newsletter ganz unten – dann löschen wir alle Daten die mit der Anmeldung zum Newsletter gespeichert wurden.
          <h3>
            MailChimp Datenschutzerklärung
          </h3>
          Wir versenden Newsletter mit MailChimp und verwenden auf dieser Webseite Funktionen des Newsletter Dienstes MailChimp der Firma The Rocket Science Group, LLC, 675 Ponce de Leon Ave NE, Suite 5000, Atlanta, GA 30308 USA um Newsletter-Anmeldungen zu erfassen.
          <h4>
            Allgemeine Informationen zu MailChimp
          </h4>
          Die Rocket Science Group LLC (MailChimp) unterhält Online-Plattformen, die es unseren Nutzern ermöglichen, in Kontakt mit ihren Abonnenten zu bleiben, in erster Linie per E-Mail. Sie erlauben Benutzern, E-Mail-Adressen und andere Informationen zum Abonnentenprofil, wie z. B. Name, physische Adresse und andere demografische Informationen, in die MailChimp Datenbank hochzuladen. Diese Informationen werden verwendet, um E-Mails zu senden und die Verwendung bestimmter anderer MailChimp-Funktionen für diese Benutzer zu ermöglichen. In Übereinstimmung mit den veröffentlichten Datenschutzrichtlinien teilt MailChimp einige Informationen mit Drittanbietern, um die Dienste bereitzustellen und zu unterstützen, die MailChimp den Nutzern anbietet. MailChimp teilt auch einige Informationen mit Werbepartnern von Drittanbietern, um die Bedürfnisse und Interessen der Nutzer besser zu verstehen, damit relevantere Inhalte und zielgerichtete Werbung für diese Nutzer und andere Nutzer bereitgestellt werden können.
          <h4>
            Newsletter Anmeldung
          </h4>
          Wenn Sie auf unserer Webseite für unseren Newsletter anmelden, werden die eingegebenen Daten bei MailChimp gespeichert.
          <h4>
            Löschung Ihrer Daten
          </h4>
          Sie können Ihre Zustimmung für den Erhalt unseres Newsletters jederzeit innerhalb der empfangenen E-Mail per Klick auf den Link im unteren Bereich entziehen. Wenn Sie sich mittels Klick auf den Abmeldelink abgemeldet haben, werden Ihre Daten bei MailChimp gelöscht.
          <h4>
            Newsletter Auswertung
          </h4>
          Wenn Sie einen Newsletter über MailChimp erhalten, werden Informationen wie IP-Adresse, Browsertyp und E-Mail-Programm gespeichert um uns Informationen zur Performance unseres Newsletters zu geben. MailChimp kann mittels den in den HTML-E-Mails integrierten Bildern namens Web Beacons (Details finden Sie auf https://kb.mailchimp.com/reports/about-open-tracking) feststellen ob die E-Mail angekommen ist, ob sie geöffnet wurde und ob Links angeklickt wurden. All diese Informationen werden auf den Servern von MailChimp gespeichert, nicht auf dieser Webseite. MailChimp ist aktiver Teilnehmer beim EU-U.S. Privacy Shield Framework wodurch der korrekte und sichere Datentransfer persönlicher Daten geregelt wird. Mehr Informationen dazu finden Sie auf https://www.privacyshield.gov/participant?id=a2zt0000000TO6hAAG. Mehr über den Einsatz von Cookies bei MailChimp erfahren Sie auf https://mailchimp.com/legal/cookies/, Informationen zum Datenschutz bei MailChimp (Privacy) können Sie auf https://mailchimp.com/legal/privacy/ nachlesen.
          <h3>
            Google Fonts Datenschutzerklärung
          </h3>
          Wir verwenden Google Fonts der Firma Google Inc. (1600 Amphitheatre Parkway Mountain View, CA 94043, USA) auf unserer Webseite. Für die Verwendung von Google-Schriftarten müssen Sie sich nicht anmelden bzw. ein Passwort hinterlegen. Weiters werden auch keine Cookies in Ihrem Browser gespeichert. Die Dateien (CSS, Schriftarten/Fonts) werden über die Google-Domains fonts.googleapis.com und fonts.gstatic.com angefordert. Laut Google sind die Anfragen nach CSS und Schriften vollkommen getrennt von allen anderen Google-Diensten. Wenn Sie ein Google-Konto haben, brauchen Sie keine Sorge haben, dass Ihre Google-Kontodaten, während der Verwendung von Google Fonts, an Google übermittelt werden. Google erfasst die Nutzung von CSS (Cascading Style Sheets) und der verwendeten Schriftarten und speichert diese Daten sicher. Wie die Datenspeicherung genau aussieht, werden wir uns noch im Detail ansehen.
          <h4>
            Was sind Google Fonts?
          </h4>
          Google Fonts (früher Google Web Fonts) ist ein interaktives Verzeichnis mit mehr als 800 Schriftarten, die die Google LLC zur freien Verwendung bereitstellt. Viele dieser Schriftarten sind unter der SIL Open Font License veröffentlicht, während andere unter der Apache-Lizenz veröffentlicht wurden. Beides sind freie Software-Lizenzen. Somit können wir sie frei verwenden, ohne dafür Lizenzgebühren zu zahlen.
          <h4>
            Warum verwenden wir Google Fonts auf unserer Webseite?
          </h4>
          Mit Google Fonts können wir auf der eigenen Webseite Schriften nutzen, und müssen sie nicht auf unserem eigenen Server hochladen. Google Fonts ist ein wichtiger Baustein, um die Qualität unserer Webseite hoch zu halten. Alle Google-Schriften sind automatisch für das Web optimiert und dies spart Datenvolumen und ist speziell für die Verwendung bei mobilen Endgeräten ein großer Vorteil. Wenn Sie unsere Seite besuchen, sorgt die niedrige Dateigröße für eine schnelle Ladezeit. Des Weiteren sind Google Fonts sogenannte sichere Web Fonts. Unterschiedliche Bildsynthese-Systeme (Rendering) in verschiedenen Browsern, Betriebssystemen und mobilen Endgeräten können zu Fehlern führen. Solche Fehler können teilweise Texte bzw. ganze Webseiten optisch verzerren. Dank des schnellen Content Delivery Network (CDN) gibt es mit Google Fonts keine plattformübergreifenden Probleme. Google Fonts unterstützt alle gängigen Browser ( Google Chrome, Mozilla Firefox, Apple Safari, Opera)  und funktioniert zuverlässig auf den meisten modernen mobilen Betriebssystemen, einschließlich Android 2.2+ und iOS 4.2+ (iPhone, iPad, iPod). Wir verwenden die Google Fonts also, damit wir unser gesamtes Online-Service so schön und einheitlich wie möglich darstellen können. Nach dem Art. 6 Abs. 1 f lit. F DSGVO stellt das bereits ein „berechtigtes Interesse“ an der Verarbeitung von personenbezogenen Daten dar. Unter „berechtigtem Interesse“ versteht man in diesem Fall sowohl rechtliche als auch wirtschaftliche oder ideelle Interessen, die vom Rechtssystem anerkannt werden.
          <h4>
            Welche Daten werden von Google gespeichert?
          </h4>
          Wenn Sie unsere Webseite besuchen, werden die Schriften über einen Google-Server nachgeladen. Durch diesen externen Aufruf werden Daten an die Google-Server übermittelt. So erkennt Google auch, dass Sie bzw. Ihre IP-Adresse unsere Webseite besucht. Die Google Fonts API wurde entwickelt, um die Erfassung, Speicherung und Verwendung von Endnutzerdaten auf das zu reduzieren, was für eine effiziente Bereitstellung von Schriften nötig ist. API steht übrigens für „Application Programming Interface“ und dient unter anderem als Datenübermittler im Softwarebereich. Google Fonts speichert CSS- und Font-Anfragen sicher bei Google und ist somit geschützt. Durch die gesammelten Nutzungszahlen kann Google die Beliebtheit der Schriften feststellen. Die Ergebnisse veröffentlicht Google auf internen Analyseseiten, wie beispielsweise Google Analytics. Zudem verwendet Google auch Daten des eigenen Web-Crawlers, um festzustellen, welche Webseiten Google-Schriften verwenden. Diese Daten werden in der BigQuery-Datenbank von Google Fonts veröffentlicht. BigQuery ist ein Webservice von Google für Unternehmen, die große Datenmengen bewegen und analysieren wollen. Zu bedenken gilt allerdings noch, dass durch jede Google Font Anfrage auch Informationen wie IP-Adresse, Spracheinstellungen, Bildschirmauflösung des Browsers, Version des Browsers und Name des Browsers automatisch an die Google-Server übertragen werden. Ob diese Daten auch gespeichert werden, ist nicht klar feststellbar bzw. wird von Google nicht eindeutig kommuniziert.
          <h4>
            Wie lange und wo werden die Daten gespeichert?
          </h4>
          Anfragen für CSS-Assets speichert Google einen Tag lang auf Ihren Servern, die hauptsächlich außerhalb der EU angesiedelt sind. Das ermöglicht uns, mithilfe eines Google-Stylesheets die Schriftarten zu nutzen. Ein Stylesheet ist eine Formatvorlage, über die man einfach und schnell z.B. das Design bzw. die Schriftart einer Webseite ändern kann. Die Font-Dateien werden bei Google ein Jahr gespeichert. Google verfolgt damit das Ziel, die Ladezeit von Webseiten grundsätzlich zu verbessern. Wenn Millionen von Webseiten auf die gleichen Schriften verweisen, werden sie nach dem ersten Besuch zwischengespeichert und erscheinen sofort auf allen anderen später besuchten Webseiten wieder. Manchmal aktualisiert Google Schriftdateien, um die Dateigröße zu reduzieren, die Abdeckung von Sprache zu erhöhen und das Design zu verbessern.
          <h4>
            Wie kann ich meine Daten löschen bzw. die Datenspeicherung verhindern?
          </h4>
          Jene Daten, die Google für einen Tag bzw. ein Jahr speichert können nicht einfach gelöscht werden. Die Daten werden beim Seitenaufruf automatisch an Google übermittelt. Um diese Daten vorzeitig löschen zu können, müssen Sie den Google-Support auf https://support.google.com/?hl=de&tid=221094977 kontaktieren. Datenspeicherung verhindern Sie in diesem Fall nur, wenn Sie unsere Seite nicht besuchen. Anders als andere Web-Schriften erlaubt uns Google uneingeschränkten Zugriff auf alle Schriftarten. Wir können also unlimitiert auf ein Meer an Schriftarten zugreifen und so das Optimum für unsere Webseite rausholen. Mehr zu Google Fonts und weiteren Fragen finden Sie auf https://developers.google.com/fonts/faq?tid=221094977. Dort geht zwar Google auf datenschutzrelevante Angelegenheiten ein, doch wirklich detaillierte Informationen über Datenspeicherung sind nicht enthalten. Es ist relativ schwierig (beinahe unmöglich), von Google wirklich präzise Informationen über gespeicherten Daten zu bekommen. Welche Daten grundsätzlich von Google erfasst werden und wofür diese Daten verwendet werden, können Sie auch auf https://www.google.com/intl/de/policies/privacy/ nachlesen.
          <h3>
            Google Maps Datenschutzerklärung
          </h3>
          Wir verwenden Google Maps der Firma Google Inc. (1600 Amphitheatre Parkway Mountain View, CA 94043, USA) auf unserer Webseite. Durch die Nutzung der Funktionen dieser Karte werden Daten an Google übertragen. Welche Daten von Google erfasst werden und wofür diese Daten verwendet werden, können Sie auf https://www.google.com/intl/de/policies/privacy/ nachlesen. Google Analytics Datenschutzerklärung Wir verwenden auf dieser Website Google Analytics der Firma Google LLC (1600 Amphitheatre Parkway Mountain View, CA 94043, USA) um Besucherdaten statistisch auszuwerten. Dabei verwendet Google Analytics zielorientierte Cookies.
          <h4>
            Cookies von Google Analytics
          </h4>
          <ul>
            <li>
              _ga
              <ul>
                <li>
                  Ablaufzeit: 2 Jahre
                </li>
                <li>
                  Verwendung: Unterscheidung der Webseitenbesucher
                </li>
                <li>
                  Beispielhafter Wert: GA1.2.1326744211.152221094977
                </li>
              </ul>
            </li>
            <li>
              _gid
              <ul>
                <li>
                  Ablaufzeit: 24 Stunden
                </li>
                <li>
                  Verwendung: Unterscheidung der Webseitenbesucher
                </li>
                <li>
                  Beispielhafter Wert: GA1.2.1687193234.152221094977
                </li>
              </ul>
            </li>
            <li>
              _gat_gtag_UA_&lt;property-id&gt;
              <ul>
                <li>
                  Ablaufzeit: 1 Minute
                </li>
                <li>
                  Verwendung: Wird zum Drosseln der Anforderungsrate verwendet. Wenn Google Analytics über den Google Tag Manager bereitgestellt wird, erhält dieser Cookie den Namen _dc_gtm_ 
                  &lt;property-id&gt;
                  .
                </li>
                <li>
                  Beispielhafter Wert: 1
                </li>
              </ul>
            </li>
          </ul>
          Nähere Informationen zu Nutzungsbedingungen und Datenschutz finden Sie unter http://www.google.com/analytics/terms/de.html bzw. unter https://support.google.com/analytics/answer/6004245?hl=de.
          <h4>
            Pseudonymisierung
          </h4>
          Unser Anliegen im Sinne der DSGVO ist die Verbesserung unseres Angebotes und unseres Webauftritts. Da uns die Privatsphäre unserer Nutzer wichtig ist, werden die Nutzerdaten pseudonymisiert. Die Datenverarbeitung erfolgt auf Basis der gesetzlichen Bestimmungen des § 96 Abs 3 TKG sowie des Art 6 EU-DSGVO Abs 1 lit a (Einwilligung) und/oder f (berechtigtes Interesse) der DSGVO.
          <h4>
            Deaktivierung der Datenerfassung durch Google Analytics
          </h4>
          Mithilfe des Browser-Add-ons zur Deaktivierung von Google Analytics-JavaScript (ga.js, analytics.js, dc.js) können Website-Besucher verhindern, dass Google Analytics ihre Daten verwendet. Sie können die Erfassung der durch das Cookie erzeugten und auf Ihre Nutzung der Website bezogenen Daten an Google sowie die Verarbeitung dieser Daten durch Google verhindern, indem Sie das unter dem folgenden Link verfügbare Browser-Plugin herunterladen und installieren: https://tools.google.com/dlpage/gaoptout?hl=de
          <h3>
            Rechte laut Datenschutzgrundverordnung
          </h3>
          Ihnen stehen laut den Bestimmungen der DSGVO und des österreichischen Datenschutzgesetzes (DSG) grundsätzlich die folgende Rechte zu:
          <ul>
            <li>
              Recht auf Berichtigung (Artikel 16 DSGVO)
            </li>
            <li>
              Recht auf Löschung („Recht auf Vergessenwerden“) (Artikel 17 DSGVO)
            </li>
            <li>
              Recht auf Einschränkung der Verarbeitung (Artikel 18 DSGVO)
            </li>
            <li>
              Recht auf Benachrichtigung – Mitteilungspflicht im Zusammenhang mit der Berichtigung oder Löschung personenbezogener Daten oder der Einschränkung der Verarbeitung (Artikel 19 DSGVO)
            </li>
            <li>
              Recht auf Datenübertragbarkeit (Artikel 20 DSGVO)
            </li>
            <li>
              Widerspruchsrecht (Artikel 21 DSGVO)
            </li>
            <li>
              Recht, nicht einer ausschließlich auf einer automatisierten Verarbeitung — einschließlich Profiling — beruhenden Entscheidung unterworfen zu werden (Artikel 22 DSGVO)
            </li>
          </ul>
          <br />
          Wenn Sie glauben, dass die Verarbeitung Ihrer Daten gegen das Datenschutzrecht verstößt oder Ihre datenschutzrechtlichen Ansprüche sonst in einer Weise verletzt worden sind, können Sie sich bei der Aufsichtsbehörde beschweren, welche in Österreich die Datenschutzbehörde ist, deren Webseite Sie unter https://www.dsb.gv.at/ finden.
        </div>
      </section>
    </div>
  </Datenschutz>
</div>
`;
